### PR TITLE
New Spices Cheat Sheet

### PR DESCRIPTION
**Pull request by shahbaz17**
---
https://duck.co/ia/view/indian_spices_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@shahbaz17 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by swapagarwal** [comment]
The name "spices cheat sheet" doesn't have any correlation with the Hindi language. It'd be better if:
1. More languages are included in spices, or
2. Things like spices are included in hindi translation [here](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/json/language/hindi.json)

**Comment by shahbaz17** [comment]
thanks @swapagarwal for your feedback..
This is my first attempt to create cheat sheet.
"Spices in Hindi cheat sheet" will this be perfect ?

**Comment by swapagarwal** [comment]
I think cheat sheets should be as generic as possible, i.e. complete information about that topic in one place. So, a spices cheat sheet can cater to various languages. Obviously, it's a community effort so everybody can (and will) collaborate to make it complete (read awesome).
About the design (keeping spices in every language separately or a new cheat sheet), let's wait for someone experienced as I'm also new here.

**Comment by moollaza** [comment]
@shahbaz17 Thanks for contributing and welcome to DuckDuckHack!

@swapagarwal thanks for the feedback.

I think we should keep this as a separate cheat sheet for now. The languages cheat sheets are all fairly consistent about the phrases they currently cover and I don't think Spice names really belong in there.

Having a cheat sheet that provides the names of spices in different languages is an interesting idea, but it would quickly become too much content for a single cheat sheet. It also opens up the doors to many other things (fruits, vegetables, etc) and I'm not sure where we'd draw the line. I don't think we'd want hundreds of cheat sheets that translate words, because most of that could probably be generated by a translation API.

Let's rename this to "Hindi Spice Names", which is a little more specific/suggestive of the content.

@shahbaz17 you'll need to change the `ID` on your `in_hindi_cheat_sheet` to `hindi_spice_names`. Then you'll need to update the `"id":` in the JSON to be `hindi_spice_names`. The filename for your cheat sheet will need to be: `hindi-spice-names.json`

@shahbaz17 also, let's add in more spice names to fill this out!

**Comment by shahbaz17** [comment]
Thanks @moollaza for this nice suggestion.
So should i close this pull request and open a new one with all the changes ?

**Comment by swapagarwal** [comment]
@moollaza :+1: for drawing the line, I too was thinking the same.

**Comment by moollaza** [comment]
@shahbaz17 sorry for the delay, I see you've updated this to be a new cheat sheet.

I've changed the IA Page ID to `indian_spices_cheat_sheet` (which now matches the article name on Wikipedia).

Please update the `id` in the JSON to match this. Also, please change the filename to `indian-spices.json`

After that this should be able to deploy to Beta for final testing + merge

File: share/goodie/cheat_sheets/json/hindi-spice-names.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    "id": "hindi_spice_names",
+    "name": "Hindi Spice Names",
+    "description": "Hindi names of spices.",
+
+    "metadata": {
+        "sourceName": "Wikipedia",
+        "sourceUrl" : "https://en.wikipedia.org/wiki/List_of_Indian_spices"
+    },
+
+    "aliases": [
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/hindi-spice-names.json]: Let's remove these aliases -- "in hindi" means that the query "in hindi cheat sheet" would show this, which it really shouldn't

File: share/goodie/cheat_sheets/json/in_hindi.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/in_hindi.json]: Please remove this file -- we only need the one JSON file

**Comment by shahbaz17** [comment]
Started a new PR https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2717

Had some problem in codio.
So started it from fresh.

@moollaza Closing this PR. Please follow the above PR

**Comment by moollaza** [comment]
> @moollaza Closing this PR. Please follow the above PR

Thanks :+1:

**Pull request closed without merging**